uk.co.compucorp.civicrm.tasksassignments/tasksassignments.php

```diff
@@ -258,6 +258,98 @@ function tasksassignments_civicrm_permission(&$permissions) {
 }
 
 /**
+ * Implements hook_civicrm_navigationMenu().
+ *
+ * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_navigationMenu/
+ */
+function tasksassignments_civicrm_navigationMenu(&$params) {
+  _tasksassignments_moveCiviCaseAdminSubMenuItemsUnderTaskAdminSubMenu($params);
+}
+
+/**
+ * Moves some of the items of the "Administer > Civi Case" sub menu under the "Administer > Tasks" sub menu
+ *
+ * @param array $params
+ */
+function _tasksassignments_moveCiviCaseAdminSubMenuItemsUnderTaskAdminSubMenu(&$params) {
+  $administerMenuItems = &_tasksassignments_getAdministerMenuItems($params);
```

Comment:
functions better to be defined near to where they first use, for example **_tasksassignments_getAdministerMenuItems()** is the first function used here so it should be defined directly below **_tasksassignments_moveCiviCaseAdminSubMenuItemsUnderTaskAdminSubMenu()**

and not so far below it to avoid much jumping between lines while reading the code. The same applies for other functions defined here.